messaging/src/main/java/org/axonframework/messaging/annotation/ParameterResolverFactory.java

```diff
@@ -52,5 +52,5 @@
      * @param parameterIndex The index of the parameter to return a ParameterResolver for
      * @return a suitable ParameterResolver, or {@code null} if none is found
      */
-    ParameterResolver createInstance(Executable executable, Parameter[] parameters, int parameterIndex);
+    ParameterResolver<?> createInstance(Executable executable, Parameter[] parameters, int parameterIndex);
```

Comment:
Strictly speaking, this is a backward-incompatible change...

Reply:
Of course...I'll place this "warning inducing issue" back in then.